clippy_lints/src/attrs.rs

```diff
@@ -407,20 +407,28 @@ declare_clippy_lint! {
     /// Checks for `#[cfg(features = "...")]` and suggests to replace it with
     /// `#[cfg(feature = "...")]`.
     ///
+    /// It also checks if `cfg(test)` was misspelled.
+    ///
     /// ### Why is this bad?
-    /// Misspelling `feature` as `features` can be sometimes hard to spot. It
+    /// Misspelling `feature` as `features` or `test` as `tests` can be sometimes hard to spot. It
     /// may cause conditional compilation not work quietly.
     ///
+    /// Same goes for misspelling `test`.
```

Comment:
This should be removed, we already mentioned `test` and `tests` in the earlier paragraph

Reply:
:+1: